pyrs/peaks/peak_collection.py

```diff
@@ -180,7 +180,8 @@ def get_dspacing_center(self):
             A two-item tuple containing the peak center and its uncertainty.
         """
         effective_values, effective_errors = self.get_effective_params()
-        theta_center_value, theta_center_error = effective_values['Center'] / 2, effective_errors['Center'] / 2
+        theta_center_value = np.array([value['Center'] / 2 for value in effective_values])
```

Comment:
Use implicit looping with numpy. Also, multiply is (generally) faster than divide
```suggestion
        theta_center_value = 0.5 * value['Center']
```